Diagnostics/Get-NetView.PS1

```diff
@@ -1556,23 +1554,42 @@ function Environment {
     }
 } # Environment()
 
-function EnvDestroy {
+function EnvSetup {
     [CmdletBinding()]
     Param(
-        [parameter(Mandatory=$true)] [String] $OutDir
+        [parameter(Mandatory=$false)] [String] $OutputDirectory,
+        [parameter(Mandatory=$true)] [String] $WorkingDirectoryName
     )
-    If (Test-Path $OutDir) {
-        Remove-Item $OutDir -Recurse
+
+    # Process/Validate $OutputDirectory
+    function NormalizeOutDir {
+        if ($OutputDirectory) {
+            if (Test-Path $OutputDirectory) {
+                return (Resolve-Path $OutputDirectory).Path # full path
+            } else {
+                Throw "Get-NetView: The directory '$OutputDirectory' does not exist."
+            }
+        } # else $OutputDirectory is undefined
+        return [Environment]::GetFolderPath("Desktop") + "\" 
+    } # NormalizeOutDir ()
+
+    # Calculate working directory
+    $workDir = Join-Path $(NormalizeOutDir) $WorkingDirectoryName
+
+    # Delete working directory (and all of its contents) if it already exists
+    If (Test-Path $workDir) {
+        Remove-Item $workDir -Recurse
```

Comment:
I put everything here because I didn't like the idea of `Remove-Item $workDir -Recurse` being separated from the definition of `$workDir`, especially now that user input involved.